models/ed/tests/test.write.restart.ED2.R

```diff
@@ -0,0 +1,33 @@
+library(PEcAn.ED2)
```

Comment:
Again, is this a test that is meant to be included in this PR? It looks like a hackathon spesific script